components/camera/camera.go

```diff
@@ -189,20 +201,31 @@ func NewVideoSourceFromReader(
 		}
 	}
 	return &videoSource{
-		videoCodecStreamSource: videoCodecStreamSource,
-		system:                 actualSystem,
-		videoSource:            vs,
-		videoStream:            gostream.NewEmbeddedVideoStream(vs),
-		actualSource:           reader,
-		imageType:              imageType,
+		rtpPassthroughSource: rtpPassthroughSource,
+		system:               actualSystem,
+		videoSource:          vs,
+		videoStream:          gostream.NewEmbeddedVideoStream(vs),
+		actualSource:         reader,
+		imageType:            imageType,
 	}, nil
 }
 
-func (vs *videoSource) VideoCodecStreamSource(ctx context.Context) (VideoCodecStreamSource, error) {
-	if vs.videoCodecStreamSource != nil {
-		return vs.videoCodecStreamSource, nil
+func (vs *videoSource) SubscribeRTP(
```

Comment:
This is needed for cameras that are wrapped in `videoSource` to still implement `rtppassthrough.SubscriptionID`